### PR TITLE
Various updates to House things

### PR DESCRIPTION
**Pull request by LtRipley36706**

### CODE REVIEW

File: Source/ACE.Server/Managers/PlayerManager.cs [hunk 1]
```diff
@@ -389,31 +389,31 @@ public static IPlayer FindByName(string name, out bool isOnline)
...
+        /// This will return null if the guid was not found.
         /// </summary>
         public static IPlayer FindByGuid(ObjectGuid guid)
         {
             return FindByGuid(guid, out _);
         }
 
         /// <summary>
-        /// This will return null of the guid was not found.
+        /// This will return null oi the guid was not found.
```
Comment (gmriggs) [inline Source/ACE.Server/Managers/PlayerManager.cs]: typo

File: Source/ACE.Server/Network/Structure/AppraiseInfo.cs [hunk 2]
```diff
@@ -155,6 +155,60 @@ public void BuildProfile(WorldObject wo, Player examiner, bool success = true)
...
+                    var discardInts = PropertiesInt.Where(x => x.Key != PropertyInt.HouseStatus && x.Key != PropertyInt.HouseType && x.Key != PropertyInt.MinLevel && x.Key != PropertyInt.MaxLevel && x.Key != PropertyInt.AllegianceMinLevel && x.Key != PropertyInt.AllegianceMaxLevel).Select(x => x.Key).ToList();
+                    foreach (var key in discardInts)
+                        PropertiesInt.Remove(key);
+                }
+
+                if (slumLord.HouseRequiresMonarch)
+                    longDesc += "You must be a monarch to purchase and maintain this dwelling.\n";
+
+                if (slumLord.AllegianceMinLevel.HasValue)
+                    longDesc += $"Restricted to characters of allegiance rank {slumLord.AllegianceMinLevel.Value} or greater.\n";
```
Comment (gmriggs) [inline Source/ACE.Server/Network/Structure/AppraiseInfo.cs]: should take into account the possible server override value

File: Source/ACE.Server/WorldObjects/House.cs [hunk 3]
```diff
@@ -245,7 +245,7 @@ public override void UpdateLinkProperties(WorldObject wo)
             SetLinkProperties(wo);
         }
 
-        public bool IsApartment => HouseType != null && HouseType.Value == ACE.Entity.Enum.HouseType.Apartment;
+        public bool IsApartment => HouseType == HouseType.Apartment;
```
Comment (gmriggs) [inline Source/ACE.Server/WorldObjects/House.cs]: is it safe to remove this null check?

Comment (LtRipley36706) [reply Source/ACE.Server/WorldObjects/House.cs]: HT is never null

**Pull request merged**